lightly/api/api_workflow_selection.py

```diff
@@ -1,9 +1,10 @@
+from __future__ import annotations
```

Comment:
This is required to not get an error for unknown `NDArray` type hint if we are not type checking.